Lib/wsgiref/headers.py

```diff
@@ -181,4 +183,10 @@ def add_header(self, _name, _value, **_params):
             else:
                 v = self._convert_string_type(v)
                 parts.append(_formatparam(k.replace('_', '-'), v))
-        self._headers.append((self._convert_string_type(_name), "; ".join(parts)))
+        header = self._convert_string_type(_name)
```

Comment:
Can you rename the variable to header_name (or maybe name, I  didn't check if it's already used)?